src/main/java/Todo.java

```diff
@@ -6,4 +6,13 @@ public Todo (String description) {
         super (description);
     }
 
+    @Override
+    public String toString() {
+        if (super.isDone) {
```

Comment:
can actually leave out the super keyword since the substitutability principle states that a todo can be replaced with a task